LibraBFT/Yasm/Properties.agda

```diff
@@ -78,6 +78,12 @@ module LibraBFT.Yasm.Properties
    trans (sym (vp-ec-≡ vp1))
          (trans (AE.lookup-𝓔s-injective 𝓔s (vp-epoch vp1) (vp-epoch vp2) parts≡)
                 (vp-ec-≡ vp2))
+ postulate
```

Comment:
Please add a `TODO-1 : prove it` comment (see the [style guide](https://github.com/oracle/bft-consensus-agda/blob/main/STYLE.md#assumptions-warnings-and-overrides)).  However, I think it's reasonable to keep this as a postulate for now as it is likely to become obsolete soon, when the moral equivalent of `ValidSenderForPK` will become an application-provided module parameter, so assuming we still need this property, it will need to be provided by the application too.